.changes/unreleased/BUG FIXES-20240118-093314.yaml

```
@@ -0,0 +1,5 @@
+kind: BUG FIXES
+body: 'schema: Remove Framework-specific implementation details from plan modifiers'
```

Since there's only two, I think we should just call them out 😄 

```suggestion
body: 'schema: Remove Framework-specific `requires_replace` and `use_state_for_unknown` plan modifiers from specification which are unused during code generation, use custom plan modifiers instead'
```